samples/GoogleHangoutsChat.gs

```diff
@@ -0,0 +1,90 @@
+var PRIVATE_KEY =
+    '-----BEGIN PRIVATE KEY-----\n...\n-----END PRIVATE KEY-----\n';
+var CLIENT_EMAIL = '...';
+
+/**
+ * Authorizes and makes a request to the Hangouts Chat API
+ * for sending a text message to all DM room bot has been added.
+ * Best before to use service account is to do the Google CodeLabs
+ * https://codelabs.developers.google.com/codelabs/chat-apps-script/
+ */
+function sendPushMessage() {
+  var service = getChatbotService();
+  if (service.hasAccess()) {
+    // We retrieve all the spaces bot has been added
+    var url = 'https://chat.googleapis.com/v1/spaces';
+    var response = UrlFetchApp.fetch(url, {
+      headers: {
+        Authorization: 'Bearer ' + service.getAccessToken()
+      }
+    });
+    var rep = JSON.parse(response.getContentText());
+    if (rep.spaces && rep.spaces.length > 0) {
+      for (var i = 0; i < rep.spaces.length; i++) {
+        var space = rep.spaces[i];
+        if (space.type == 'DM') {
+          // We send message only to Direct Message room.
+          var url = 'https://chat.googleapis.com/v1/'+space.name+'/messages';
+          var options = {
+            method: 'POST',
+            contentType: 'application/json',
+            headers: {
+              Authorization: 'Bearer ' + service.getAccessToken()
+            },
+            payload: JSON.stringify({text: 'Hello world !'})
+          };
+          // We send message to the DM room
+          UrlFetchApp.fetch(url, options);
+        } else {
+          // If Type is 'ROOM' or 'TYPE_UNSPECIFIED' we don't send notification.
+        }
+      }
+    } else {
+      Logger.log('Bot is not added to any spaces');
+    }
+  } else {
+    Logger.log(service.getLastError());
+  }
+}
+
+/**
+ * Reset the authorization state, so that it can be re-tested.
+ */
+function reset() {
+  getService().reset();
```

Comment:
You are using `getService()` here, but I think you renamed the function `getChatbotService()`.